tests/e2e/service/iothub_client/e2e_iot_hub_client_test.ts

```diff
@@ -215,4 +216,34 @@ describe("mainTest", () => {
         let expectedMessage = await iothubTestCore.runTestCommand(testHubConnectionString, testDeviceInfo.deviceId, get_twin_cmd);
         let rc = await iothubTestCore.verifyTelemetryMessage(testHubConnectionString, testDeviceInfo.deviceId, expectedMessage);
     })
+
+    it("Device Writable Properties Response", async function() {
+        const payload = { "temprature" : (Math.floor(Math.random() * 10000000000).toString())};
+        const writable_properties_cmd = new e2eTestCommands.CommandTestData_WritablePropertiesResponseCommands("temprature", payload["temprature"]);
+        let expectedMessage = await iothubTestCore.runTestCommand(testHubConnectionString, testDeviceInfo.deviceId, writable_properties_cmd);
+        let rc = await iothubTestCore.verifyTelemetryMessage(testHubConnectionString, testDeviceInfo.deviceId, expectedMessage);
+        let twin = await iothubTestCore.getTwinProperties(testHubConnectionString, testDeviceInfo.deviceId);
+        console.log(twin)
+        var twinObject = JSON.parse(twin);
+        assert.ok(twinObject.reported.temprature);

```

Comment:
is this (and elsewhere) just asserting that this value exists?

Reply:
yes, this tells this object exists and the value is tested line 230and 233.